public/js/newsManagerView.js

```diff
@@ -0,0 +1,41 @@
+(function(exports){
+
+function NewsManagerView () {
+
+}
+
+NewsManagerView.prototype = {
```

Comment:
This code could possibly be simplified by separating the idea of generating the HTML (eg lines 10 to 13) from the idea of inserting the new HTML into the DOM (eg line 14).  The former can be done with little dependency-less functions that take data and return HTML.